packages/client/hmi-client/src/workflow/ops/calibrate-ensemble-ciemss/calibrate-ensemble-ciemss-operation.ts

```diff
@@ -5,8 +5,8 @@ import { ChartConfig } from '@/types/SimulateConfig';
 import type { EnsembleModelConfigs } from '@/types/Types';
 
 export interface EnsembleCalibrateExtraCiemss {
-	numSamples: number;
-	totalPopulation: number;
+	solverMethod: string;
+	numParticles: number;
 	numIterations: number;
 }
```

Comment:
What prompted the naming change? 
What does `numParticles` imply?

Reply:
Im not certain - I am talking to Five and Nelson

Do you mean a document for users? like what will be in top right?
<img width="1460" alt="image" src="https://github.com/DARPA-ASKEM/terarium/assets/17088680/4156f1de-dcd1-4d98-9851-fdd86a65d76c">

Or do you mean a doc in the .ts file?